code/obj/machinery/door/airlock.dm

```diff
@@ -361,6 +375,31 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	icon_base = "sec_glass"
 	req_access = list(access_security)
 
+/obj/machinery/door/airlock/pyro/glass/med 
+	icon_state = "med_glass_closed"
+	icon_base = "med_glass"
+	req_access = null
```

Comment:
👍

Reply:
IMO all doors should have req_access be null, and the onus would be on the mapper to specify access for each door on their map. But since it's already like this and changing it would break existing maps idk. I don't like it but that's how it be.